cpp/Integri/XsdComparator/XsdComparator.cpp

```diff
@@ -0,0 +1,219 @@
+#include "XsdComparator.h"
+#include <iostream>
+#include <stdio.h>
+#include <tchar.h>
+#include <windows.h>
+#include <atlbase.h>
+#include <comutil.h>
+
+using namespace std;
+
+XsdComparator::XsdComparator(Reporter* reporter)
+{
+	this->reporter = reporter;
```

Comment:
Here I recommend using the ctor initializer. It would look as follows:
`XsdComparator::XsdComparator(Reporter* reporter) : reporter(_reporter) {}`
Notice, that you will have to rename the ctor parameter. This is a usual practice.
The advantage of doin it this way is that in your case the following will happen:

1. Class is created and all members are initialized to their default values in the order of declaration in the header.
2. Ctor body is executed and the default values are replaced.

With the class initializer you replace the default initialization with your values. It's a small optimization but it is a common practice.